Polygon_repair_2/examples/Polygon_repair_2/write_labeled_triangulation.cpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <iomanip>
+
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/Polygon_repair_2/Polygon_repair_2.h>
+#include <CGAL/IO/WKT.h>
+
+using Kernel = CGAL::Exact_predicates_inexact_constructions_kernel;
+using Point_2 = Kernel::Point_2;
+using Polygon_2 = CGAL::Polygon_2<Kernel>;
+using Polygon_with_holes_2 = CGAL::Polygon_with_holes_2<Kernel>;
+using Multipolygon_with_holes_2 = CGAL::Multipolygon_with_holes_2<Kernel>;
+using Polygon_repair_2 = CGAL::Polygon_repair_2::Polygon_repair_2<Kernel>;
+
+int main(int argc, char* argv[]) {
+
+  // std::ifstream ifs("/Users/ken/Downloads/180927.wkt");
+  // std::ofstream ofs("/Users/ken/Downloads/2.geojson");
+
+  // std::ifstream ifs("/Users/ken/Downloads/2018418.wkt");
+  // std::ofstream ofs("/Users/ken/Downloads/1.geojson");
+
+  std::ifstream ifs("../../test/Polygon_repair_2/data/in/nesting-spike.wkt");
+  std::ofstream ofs("/Users/ken/Downloads/triangulation.geojson");
+
+  std::string in;
+  std::getline(ifs, in);
+  std::istringstream iss(in);
+  Polygon_repair_2 pr;
+
+  if (in.find("POLYGON") == 0) {
+    Polygon_with_holes_2 p;
+    if (in != "POLYGON()") { // maybe should be checked in WKT reader
```

Comment:
Do you mean it might be a bug in CGAL or boost?

Reply:
It's an unhandled case of invalid input I found while doing tests, but maybe the check shouldn't be there at all. Arguably, it's more of a syntax error than an invalid geometry since the polygon has no outer ring.

What is a bit weird IMO is that `MULTIPOLYGON()` is handled fine but `POLYGON()` isn't.